src/Pyz/Yves/CheckoutPage/Theme/default/templates/page-layout-order-submitted/page-layout-order-submitted.twig

```diff
@@ -26,3 +26,22 @@
         {% endif %}
     </div>
 {% endblock %}
+
```

Comment:
if this page will be extended from `{% extends template('page-layout-order-submitted.', '@SprykerShop:CheckoutPage') %}` can we remove this copy-paste block?

Reply:
yes we can. But this block is not extendeds from `page-layout-order-submitted`

Reply:
discussed, the change will break the UI.